ADAL/tests/integration/ios/ADBrokerIntegrationTests.m

```diff
@@ -193,6 +193,90 @@ - (void)testBroker_whenSimpleAcquireToken_shouldSucceed
     XCTAssertEqualObjects([tokenCache getFRT:authority], @"i-am-a-refresh-token");
 }
 
+- (void)testBroker_whenSimpleAcquireToken_failsWithProtectionRequiredError
```

Comment:
nit: we use the naming convention `test..._when..._should...` in the project. I guess we could rename the function to `testBroker_whenSimpleAcquireToken_shouldFailsWithProtectionRequiredError`